content/docs/user-guide/dvc-files.md

```diff
@@ -20,8 +15,13 @@ Both `.dvc` files and `dvc.yaml` use human-friendly YAML 1.2 schemas, described
 below. We encourage you to get familiar with them so you may create, generate,
 and edit them on your own.
 
-Both the internal directory and these metafiles should be versioned with Git (in
-Git-enabled <abbr>repositories</abbr>).
+These metafiles should be versioned with Git (in Git-enabled
+<abbr>repositories</abbr>).
+
+See the [internals guide](/doc/user-guide/dvc-internals) for the contents of the
+`.dvc/` directory.
```

Comment:
This link is meant to preserve/transfer the SEO ranking for the now separate guide on internals.